docs/src/content/en/docs/mastra-cloud/dashboard.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Understanding the Mastra Cloud Dashboard
+description: Details of each feature available in Mastra Cloud
+---
+
+import { MastraCloudCallout } from '@/components/mastra-cloud-callout'
+
+# Navigating the Dashboard
+
+This page explains how to navigate the Mastra Cloud dashboard, where you can configure your project, view deployment details, and interact with agents and workflows using the built-in [Playground](/docs/mastra-cloud/dashboard#playground).
+
+<MastraCloudCallout />
+
+## Overview
+
+The **Overview** page provides details about your application, including its domain URL, status, latest deployment, and connected agents and workflows.
+
+![Project dashboard](/image/mastra-cloud/mastra-cloud-project-dashboard.jpg)
+
+Key features:
+
+- **Created Date**: Timestamp for when the project was first created
+- **Domains**: URLs you can use to access the deployed application
+- **Status**: Current deployment state, e.g.,success, failed or deploying
+- **Branch**: Git branch used for deployment, usually `main`
+- **Environment Variables**: Keys and secrets used by your application
+- **Workflows**: Detected workflows included in the application
+- **Agents**: Detected agents included in the application
+- **Storage Usage**: Current usage of built-in storage
+
+## Deployments
+
+The **Deployments** page shows recent builds, their status, and gives you quick access to detailed build logs. Click any row to view more information about a specific deployment.
+
+![Dashboard deployment](/image/mastra-cloud/mastra-cloud-dashboard-deployments.jpg)
+
+Key features:
+
+- **Title**: Auto-generated from the commit hash
+- **Status**: Current deployment state, e.g. success, failed or deploying
+- **Branch**: Git branch used for this deployment
+- **Commit**: Full Git commit hash
+- **Updated At**: When the deployment was created
```

Comment:
Please write like a human! For example, in this case, it should say:

"This includes title (auto-generated from the commit hash), status (e.g. success, failed deploying), branch/commit and updated at."